src/test/java/org/cactoos/io/TailOfTest.java

```diff
@@ -40,79 +40,82 @@
     public void tailsOnLongStream() throws Exception {
         final int size = 4;
         final byte[] bytes = this.generate(size);
-        MatcherAssert.assertThat(
-            new BytesOf(
-                new TailOf(
-                    new InputOf(new BytesOf(bytes)),
-                    size - 1
-                )
-            ).asBytes(),
-            Matchers.equalTo(Arrays.copyOfRange(bytes, 1, bytes.length))
-        );
+        new Assertion<>(
+        "Must tail on Long stream",
```

Comment:
@mohamednizar Let's change this assertion failure message to "Must return last part of Long stream", making it
somewhat closer to `tail` description on linx manual.